app/build.gradle

```diff
@@ -51,6 +51,7 @@ android {
 
             kotlinOptions {
                 freeCompilerArgs = [
+                    "-Xcontext-receivers",
```

Comment:
For using context receivers feature, see https://github.com/Kotlin/KEEP/blob/master/proposals/context-receivers.md